hs/src/Reach/AST/DLBase.hs

```diff
@@ -1165,3 +1159,13 @@ type DLViews = InterfaceLikeMap IType
 type DLAPIs = InterfaceLikeMap (SLPart, IType)
 
 type DLEvents = InterfaceLikeMap [DLType]
+
+arraysLength :: [DLArg] -> Integer
+arraysLength arrays = do
+  let sizes = map (\x -> case argTypeOf x of
+                                 T_Array _ n -> Just n
```

Comment:
There's already a function that does this

Reply:
Or if I'm remember wrong... there should be be :)

Reply:
`arrTypeLen` is what I'm thinking of. You don't care about the `Just` because you're just going to do an impossible later